app/decorators/changelog_entry_decorator.rb

```diff
@@ -1,6 +1,16 @@
 # frozen_string_literal: true
 
 class ChangelogEntryDecorator < ApplicationDecorator
+  class << self
+    def authors_options
+      User.pluck(:id, :name)
```

Comment:
```suggestion
      User.pluck(:id, :to_s)
```

To be tested to confirm it works